### ISSUE TITLE
Check status of OTR session from hexchat?

### ISSUE THREAD

**Issue by petterreinholdtsen** [TingPing/hexchat-otr#16]
Is there a way to check the status of an OTR session from within hexchat?  I would like to know if a session is going to be encrypted and what the fingerprints for the session are.  Something like /otr status or similar.  No idea what other OTR clients are doing, but perhaps there is some well known /otr command used elsewere that could be implemented?

**Comment by petterreinholdtsen** [comment]
According to https://github.com/cryptodotis/irssi-otr , the irssi support provides '/otr info' reporting something similar.

**Comment by dramb-lys** [comment]
I don't think this is an enhancement. I think this is necessary for the user to trust OTR.

See https://github.com/cryptodotis/irssi-otr for a fairly nice implementation of this.

**Comment by lelutin** [comment]
I agree that this is sorely missing. the `/otr info` command would be great to have, but even better in addition to the command it would be to have a visual flag/logo somewhere that always shows a conversation's status.

**Comment by petterreinholdtsen** [comment]
So I guess all that is missing is for someone to implent it and send a pull request?

**Comment by devnull-hub-lab** [comment]
I'm implementing /otr status, when fully tested, I will send a PR.

**Comment by petterreinholdtsen** [comment]
[Rafael Grether]
> I'm implementing /otr status, when fully tested, I will send a PR.

I am glad to hear there is still some development taking place.
-- 
Happy hacking
Petter Reinholdtsen
